### PR TITLE
ES with docker-compose

### PR DESCRIPTION
**Pull request by kazesberger**
adds an ES container to docker-compose. 
solves issue #2

### CODE REVIEW

**Comment by boernd** [comment]
Is the 2nd commit intended?

File: src/riemann_history/core.clj [hunk 1]
```diff
@@ -66,3 +66,12 @@
   (let [client (es/client {:hosts [(:connect config "http://localhost:9200")]})]
     (every! (:interval config 86400) 10 (query client config))))
 
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;; EOF ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+
+(comment
```
Comment (boernd) [inline src/riemann_history/core.clj]: Can we remove that?

Comment (kazesberger) [reply src/riemann_history/core.clj]: yes we can! :)

**Comment by kazesberger** [comment]
still under heavy dev (created PR in case some1 wanted to make progress using local ES setup while i'm on vacation)

**Comment by kazesberger** [comment]
fixes #2

**Pull request merged**